common/src/main/kotlin/io/homeassistant/companion/android/common/compose/composable/HAButtons.kt

```diff
@@ -253,17 +285,23 @@ private fun RowScope.ButtonContent(
     text: String,
     prefix: @Composable (BoxScope.() -> Unit)?,
     suffix: @Composable (BoxScope.() -> Unit)?,
+    textOverflow: TextOverflow,
+    maxLines: Int,
 ) {
     ButtonDecorator(ButtonDecoratorType.PREFIX, prefix)
     Text(
         text = text,
         style = HATextStyle.Button,
+        maxLines = maxLines,
+        overflow = textOverflow,
```

Comment:
It annoys me that this order doesn't match the function arguments' order 😅

Reply:
I'll address it 👍. To be fair I'm still struggling with myself adhering to a convention for the parameters order outside of the practical cases like a lambda at the end or default at the end.